master/internal/api_workspace_intg_test.go

```diff
@@ -521,3 +522,29 @@ func TestAuthzWorkspaceGetThenActionRoutes(t *testing.T) {
 		require.Equal(t, expectedErr.Error(), curCase.IDToReqCall(id).Error())
 	}
 }
+
+func TestWorkspaceHasModels(t *testing.T) {
+	// set up a dB and api server to use for integration testing
+	require.NoError(t, etc.SetRootPath("../static/srv"))
+	pgDB, cleanup := db.MustResolveNewPostgresDatabase(t)
+	defer cleanup()
+	db.MustMigrateTestPostgres(t, pgDB, "file://../static/migrations")
+	api, _, ctx := setupAPITest(t, pgDB)
+
+	// create workspace for test
+	workspaceName := "test-workspace"
+	resp, err := api.PostWorkspace(ctx, &apiv1.PostWorkspaceRequest{Name: workspaceName})
+	require.NoError(t, err)
+
+	// confirm workspace does not have any models
+	exists, err := api.workspaceHasModels(ctx, resp.Workspace.Id)
+	assert.False(t, exists)
```

Comment:
```assert.False``` will return a boolean if the assertion is true or false? So this line could never fail the test?

I think we want to replace all ```assert.``` with ```require.```?

Reply:
`assert.False` asserts that the specified value is false. It passes if `exists` is false and does not if `exists` is true. 

Although, I notice that other tests use require instead of assert; I'll change this to be consistent. I found a nice explanation for the different uses of `assert` and `require`:
``` Using assert will allow the developer to learn about failures on different tests as they occur but it will not halt the rest of the tests execution. But using require will stop the tests on the first failure, providing a fail-fast approach.```